### PR TITLE
Update upgrade-distribution-lists.md

### PR DESCRIPTION
**Pull request by anirudhyadavMS**
@MikePlumleyMSFT  this reflects the deprecation plan for the feature

### CODE REVIEW

File: microsoft-365/admin/manage/upgrade-distribution-lists.md [hunk 1]
```diff
@@ -27,10 +27,13 @@ description: "Learn how to upgrade one or many distribution lists to Microsoft 3
 
 # Upgrade distribution lists to Microsoft 365 Groups in Exchange Online
 
+**_NOTE:_**  This functionality is going to be deprecated by 1st Feb, 2023.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/upgrade-distribution-lists.md]:
```suggestion
> [!NOTE]
> This functionality will be deprecated by February 1, 2023.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anirudhyadavMS, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by anirudhyadavMS** [comment]
@MikePlumleyMSFT Please merge this.

**Comment by MikePlumleyMSFT** [comment]
@kwekuako

File: microsoft-365/admin/manage/upgrade-distribution-lists.md [hunk 2]
```diff
@@ -28,10 +28,14 @@ description: "Learn how to upgrade one or many distribution lists to Microsoft 3
 
 # Upgrade distribution lists to Microsoft 365 Groups in Exchange Online
 
+> [!Important]
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/upgrade-distribution-lists.md]:
```suggestion
> [!IMPORTANT]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@anirudhyadavMS, kindly commit the suggested change. Many thanks.

File: microsoft-365/admin/manage/upgrade-distribution-lists.md [hunk 3]
```diff
@@ -28,10 +28,14 @@ description: "Learn how to upgrade one or many distribution lists to Microsoft 3
 
 # Upgrade distribution lists to Microsoft 365 Groups in Exchange Online
 
+> [!IMPORTANT]
+> This functionality will be deprecated by February 1, 2023. For more information, see [Deprecating Conversion of Distribution Lists to Microsoft 365 Groups - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/exchange-team-blog/deprecating-conversion-of-distribution-lists-to-microsoft-365/ba-p/3668215)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/upgrade-distribution-lists.md]:
```suggestion
> This functionality will be deprecated by February 1, 2023. For more information, see [Deprecating Conversion of Distribution Lists to Microsoft 365 Groups - Microsoft Community Hub](https://techcommunity.microsoft.com/t5/exchange-team-blog/deprecating-conversion-of-distribution-lists-to-microsoft-365/ba-p/3668215).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @anirudhyadavMS, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**